styles/styles.css

```diff
@@ -459,9 +554,13 @@
     --col-width: calc(100% / 12);
   }
 
-  /*[class*="col-"] {*/
-  /*  width: 50%;*/
-  /*}*/
+  .villa {
```

Comment:
it's not good way to give name for classes like villa, hostel and  etc. 
also as you can see you duplicate the same code in numerous classes, better to create some class with abstract name and use it

Reply:
you can use such names as card, block-item, etc. with prefix with name of main block like advantages

Reply:
Hostel, vila  in name it's bad idea because you will get a lot of daa from backend in future which will be parsed on  youe design, so it could no have such items as villa or it's can be a lot of villas an etc. Thats why, use more abstract names